apps/aecore/src/aec_block_key_candidate.erl

```diff
@@ -17,19 +16,13 @@
 -include("blocks.hrl").
 
 %% -- API functions ----------------------------------------------------------
--spec create(aec_blocks:block() | aec_blocks:block_header_hash(),
-             aec_keys:pubkey()) ->
+-spec create(aec_blocks:block(), aec_keys:pubkey()) ->
```

Comment:
OK, and you can do it, because you added top block to conductor's state

Reply:
This saves 1 db read - instead of passing the hash and reading the block from the hash, the whole block is passed.